File: WarehouseManagementWeb.Application/Dto/Input/ResourceReceipt/IncludeResourceReceiptInput.cs
namespace WarehouseManagementWeb.Application.Dto.Input.ResourceReceipt
{
    /// <summary>
    /// Класс входной модели добавления ресурсов в поступление.
    /// </summary>
    public class IncludeResourceReceiptInput
    {
        /// <summary>
        /// Id ресурса.
        /// </summary>
        public int? ResourceId { get; set; }

        /// <summary>
        /// Id единицы измерения.
        /// </summary>
        public int? MeasureUnitId { get; set; }

        /// <summary>
        /// Количество ресурса.
        /// </summary>
        public int? ResourceQuantity { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/warehouse/document-receipt/models/input/include-resource-receipt-input.ts
/**
 * Класс входной модели добавления ресурсов в поступление.
 */
export class IncludeResourceReceiptInput {
  /**
   * Id ресурса.
   */
  resourceId: number | null = null;

  /**
   * Id единицы измерения.
   */
  measureUnitId?: number | null = null;

  /**
   * Количество ресурса.
   */
  resourceQuantity: number = 0;
}